client/elements/sc-page-dictionary.html

```diff
@@ -218,24 +212,31 @@ <h3>Similar Spelling</h3>
                       }
                   },
                   // The below two arrays are just temporary mockups.
-                  dictonaryAdjacent: {
+                  dictionaryAdjacent: {
                       type: Array,
                       value: ['dhamita','dhamitvā','dhameti','dhamenta','dhamesi',
                       'dhamma','dhammakathā','dhammakathika','dhammakamma','dhammakaraka']
                   },
+                  dictionaryAdjacentArray: Array,
```

Comment:
hmm, `dictionaryAdjacent` and `dictionaryAdjacentArray` are both arrays. Do you think you could differentiate them somehow by name?

Reply:
Once it is working to import the right data, the bogus array will be eliminated and you only have "dictionaryAdjacent" left. This is just a temporary solution.

Reply:
Technically, we shouldn't commit work in progress into the main branch. If that's the way you like to work, that's fine, but ideally you'd have a separate branch that you keep updated by continuously merging with `develop`, and only create a PR once that particular piece of work is ready.

Reply:
I see. It's just that Blake will continue with it (some data from Elasticsearch need to go in) so I want him to be able to pick it up. Guess he can pick up a separate branch too. I guess I should have kept the glossary part separate from the adjacent part and committed it to two different branches.